tests/docker-compose.yml

```diff
@@ -0,0 +1,39 @@
+version: '3.3'
+services:
+  postgresdb:
+    container_name: importexport_pgdb
+    environment:
+      DB_HOST: db
+      DB_PORT: 5432
+      DB_NAME: import_export
+      IMPORT_EXPORT_POSTGRESQL_USER: ${IMPORT_EXPORT_POSTGRESQL_USER}
+      IMPORT_EXPORT_POSTGRESQL_PASSWORD: ${IMPORT_EXPORT_POSTGRESQL_PASSWORD}
+      POSTGRES_PASSWORD: ${IMPORT_EXPORT_POSTGRESQL_PASSWORD}
+    image: postgres:10
+    restart: "no"
+    ports:
+      - "5432:5432"
+    volumes:
+      - ./docker/db/:/docker-entrypoint-initdb.d/
```

Comment:
I can't see tests/docker/db/ in this PR, maybe `tests/bulk/docker/db/init-user-db.sh` was not supposed to be deleted, just moved there?

Reply:
Yes I moved `tests/bulk/docker/` down one level to `tests` because previously the docker container was used solely for bulk testing - now it can be used for tox as well, so makes sense to be present in `tests`